src/main/java/core/basesyntax/shop/service/ShopDao.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.shop.service;
+
+import core.basesyntax.shop.InsufficientGoodsException;
+import java.util.Map;
+
+public interface ShopDao {
```

Comment:
It's not a service.
Seems should be FruitStorageDao and be putted to the separate package.